README.md

```diff
@@ -441,8 +441,211 @@ userAssignedIdentity:
+          - roleName: <string>                                --RoleAssignment Name (Accepted values = "BlobDataContributor")    
+    tables:                                               --List of Tables and roleassignments
+      - name: <string>                                        --Table name
+        roleAssignments:                                      --List of roleAssignments scope to the table
+          - roleName: <string>                                --RoleAssignment Name (Accepted values = "TableDataContributor")
+```
+#### Usage examples
+The following section provides usage examples for the storage account template.
+
+##### Example 1 : Create 2 storage accounts
+
+```
+storageAccounts:
+  - name: storage01
+  - name: storage02
+
+```
+
+##### Example 2 : Create 1 storage account using large parameter set and storage firewall
+
+```
+storageAccounts:
+  - name: storage01
+    accessTier: Hot
+    location: uksouth
+    allowBlobPublicAccess: false
+    allowCrossTenantReplication: false
+    allowSharedKeyAccess: true
+    defaultToOAuthAuthentication: false
+    minimumTlsVersion: TLS1_2
+    ipRules:
+    - 82.13.86.001
+    - 82.13.86.002
+    virtualNetworkRules:
+    - "/subscriptions/<subscriptionId>/resourceGroups/<resourceGroupName>/providers/Microsoft.Network/virtualNetworks/<virtualNetworkName>/subnets/<subnetName1>"
+    - "/subscriptions/<subscriptionId>/resourceGroups/<resourceGroupName>/providers/Microsoft.Network/virtualNetworks/<virtualNetworkName>/subnets/<subnetName2>"
+    publicNetworkAccess: Enabled
+    sku:
+      name: Standard_RAGRS
+
+```
+
+##### Example 3 : Create 1 storage account and configure properties for the Storage Account BlobService
+
+```
+storageAccounts:
+  - name: storage01
+    storageAccountsBlobService:
+      changeFeed:
+        enabled: true
+        retentionInDays: 14
+      containerDeleteRetentionPolicy:
+        days: 14
+        enabled: true
+      deleteRetentionPolicy:
+        allowPermanentDelete: true
+        days: 20
+        enabled: false
+      isVersioningEnabled: true
+      restorePolicy:
+        enabled: true
+        days: 40
+
+```
+
+##### Example 4 : Create 1 storage account with 2 blob containers and 1 table with roleassignments
+
+```
+storageAccounts4:
+  - name: storage01
+    blobContainers:  
+      - name: container-01
+        roleAssignments:
+          - roleName: 'BlobDataContributor' 
+      - name: container-02
+        roleAssignments:
+          - roleName: 'BlobDataContributor'   
+    tables:  
+      - name: table01  
+        roleAssignments:
+          - roleName: 'TableDataContributor' 
+
+```
+
+##### Example 5 : Create 1 blob containers and 2 tables for the existing storage account in your team.
+
+```
+storageAccounts4:
+  - name: storage01
+    owner: "No"               --Note owner is set to 'No' to indicate storage account is already exit and own by the different service in the team.
```

Comment:
@ganeshg2491 - A couple of typos exit and own.  Should they say exists and owned.

What do you think of the below suggestion?
Note owner is set to 'No' to indicate storage account already exists and is owned by a different service in the team.

Reply:
@aadev1 : Updated